### PR TITLE
DOCS-935-encryption-icons

### PR DESCRIPTION
**Pull request by q5sys**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/en/hub/initial-setup/storage/encryption.md [hunk 1]
```diff
@@ -79,8 +79,14 @@ Must be longer than 8 characters
 
 ## Locking and Unlocking Datasets
 
+Status of a dataset can be determined based on the icon used after the name.  
+
+Dataset Unlocked Icon : <mat-icon _ngcontent-iee-c469="" role="img" fontset="mdi-set" fonticon="mdi-lock-open-variant" class="mat-icon notranslate mdi-set mdi-lock-open-variant mat-icon-no-color ng-star-inserted" aria-hidden="true"></mat-icon>
+
+Dataset Locked Icon : <mat-icon _ngcontent-iee-c469="" role="img" fontset="mdi-set" fonticon="mdi-lock" class="mat-icon notranslate mdi-set mdi-lock mat-icon-no-color ng-star-inserted" aria-hidden="true"></mat-icon>
```
Comment (Mrt134) [inline content/en/hub/initial-setup/storage/encryption.md:86]: These icons aren't showing up in the site preview. Try reworking to use this syntax instead: `<i class="material-icons" aria-hidden="true" title="<hovertext>"><iconname></i>`

**Review by Mrt134 — CHANGES_REQUESTED** [review]

File: content/en/hub/initial-setup/storage/encryption.md (hunk 1 above)
Comment (q5sys) [reply content/en/hub/initial-setup/storage/encryption.md]: Still not working.  This is the exact icon that the UI is using.  Are they using more than one Material Design Icon set?

**Comment by q5sys** [comment]
ok this is sort of fixed.  We can't use the icons the UI uses because they're using a different material design set we dont have set up on the Docs site.  So the unlock icon is not the same as whats in the UI, but its close enough at this point to explain it to the user.

**Review by Mrt134 — APPROVED** [review]
Changes look good! I've opened a ticket in the Docs project to add the other icon pack.

**Pull request merged**